content/applications/finance/accounting/fiscal_localizations/localizations/united_kingdom.rst

```diff
@@ -0,0 +1,84 @@
+==============
+United Kingdom
+==============
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the UK
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`UK - Accounting`
+     - `l10n_uk`
+     - Installing this module grants you access to:
+        - CT600-ready chart of accounts
+        - VAT100-ready tax structure
+        - InfoLogic UK counties listing
+        - a few other adaptations
+   * - :guilabel:`UK - Accounting Reports`
+     - `l10n_uk_reports`
+     - Installing this module grants you access to:
+        - Accounting reports for UK
+        - Allows sending the tax report via the MTD-VAT API to HMRC.
+
+.. tip::
+   Installing the module :guilabel:`UK - Accounting Reports` installs all two modules at
+   once.
+
+.. note::
+   Your company should be located in the UK to be able to submit HMRC reports.
+
+.. seealso::
+   - `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
+   - `Overview of Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_
+
+Making Tax Digital (MTD)
+========================
+
+Making Tax Digital is a key part of the UK government’s plans to make it easier for individuals and
```

Comment:
```suggestion
Making Tax Digital is a key part of the UK government’s plans to make it easier for individuals and
```
I think users looking for the UK localization already know what MTD is, and Odoo isn't the place for that information. IMO it is better to keep it straight to the point, the line "All VAT-registered businesses..." is much more relevant to the doc :)

Reply:
make sens!